modules/solid_mechanics/test/tests/ad_anisotropic_creep/ad_aniso_creep_integration_error.i

```diff
@@ -203,6 +194,8 @@
   growth_factor = 1.5
   cutback_factor = 0.5
   timestep_limiting_postprocessor = matl_ts_min
+  reject_large_step = true
+  reject_large_step_threshold = 0.999
```

Comment:
so this is just a friendlier setup for this step rejection?

Reply:
I don't think that is the case. There is no way to execute a userobject between restoration and when it is used to constrain a timestep. This is an extremely unique usage of a PP value, so I think it's appropriate to implement the proper logic within the object requiring the uniqueness.